collect_app/src/main/java/org/odk/collect/android/utilities/ThemeUtils.java

```diff
@@ -125,4 +125,12 @@ public int getIconColor() {
     public int getRankItemColor() {
         return getAttributeValue(R.attr.rankItemColor);
     }
+
+    /**
+     * @return Secondary color for the current {@link android.content.res.Resources.Theme}
```

Comment:
I'm not a fan of such obvious comments. Comment's should be used only if it's required to explain something. Could you remove them from the entire file?

Reply:
Yes sure @grzesiek2010!